src/Components/Authorization/src/CascadingAuthenticationState.razor

```diff
@@ -1,7 +1,7 @@
 ﻿@implements IDisposable
 @inject AuthenticationStateProvider AuthenticationStateProvider
 
-<CascadingValue TValue="Task<AuthenticationState>" Value="@_currentAuthenticationStateTask" ChildContent="@ChildContent" />
+<CascadingValue TValue="System.Threading.Tasks.Task<AuthenticationState>" Value="@_currentAuthenticationStateTask" ChildContent="@ChildContent" />
```

Comment:
@pranavkm Here's the issue I mentioned

Reply:
https://github.com/dotnet/aspnetcore/issues/39950

Reply:
This is fixed now